service-netty/src/main/java/info/bitrich/xchangestream/service/netty/NettyStreamingService.java

```diff
@@ -53,12 +38,27 @@
 import io.reactivex.Observable;
 import io.reactivex.ObservableEmitter;
 import io.reactivex.subjects.PublishSubject;
+import org.slf4j.Logger;
```

Comment:
We should really add coveo-fmt. Don't want to do it when there are a lot of outstanding PRs though. Catch-22.